### ISSUE TITLE
enforce referential integrity for employers

### ISSUE THREAD

**Issue by jhayes45** [wdjg/coccareerfairapp-server#75]
john on slack: 
"The current linestatus breaks my code because it was made using an old employer_id, so technically there's a line for a company that doesn't exist."

this is undesirable and should be avoided. options:
1) check if there's a way to do enforce referential integrity automatically in mongoose/mongodb (akin to mysql ON DELETE/ON UPDATE)
2) use a cron job that checks to make sure all lines' employers still actually exist
3) make sure that using the DELETE employers/:id route also deletes any lines for that employer.

**Comment by brianwang9100** [comment]
i vote cascade, its the cleanest

**Comment by jhayes45** [comment]
yeah I was going to look into that first and then try approaches 2/3 if
there were problems

**Comment by jhayes45** [comment]
Either way I'm not doing this until I finish the #sprint-objectives on
slack for the front end
